tutorials/uniformSampling/tutorial_uniformSampling.m

```diff
@@ -91,12 +91,17 @@
 % The CHRR algorithm is an useful tool to sample high dimentional models (bigger 
 % than 10,000 reactions). 
 %% Equipment setup
+% To perform FVA using the function |fastFVA| it is necessary to have CPLEX 
+% solver installed. In this tutorial, we will use the function |fluxVariability|. 
+% Change the variable |options.useFastFVA = 1 |to use |fastFVA|.
 
- changeCobraSolver('gurobi');
+options.useFastFVA = 0;
```

Comment:
This option `options.useFastFVA` should be an explicit boolean, i.e. `options.useFastFVA = false;`